src/unitquaternion.jl

```diff
@@ -20,17 +20,17 @@ struct QuatRotation{T} <: Rotation{3,T}
     @inline function QuatRotation{T}(w, x, y, z, normalize::Bool = true) where T
```

Comment:
It looks like this method should just forward to `QuatRotation{float(T)}(q,normalize)`, rather then replicate its functionality. Further, this uses a non-equivalent normalization due to implementation differences in `abs` (and potentially also in the implementation of division by a real divisor).

Reply:
Done!